api/base/settings/defaults.py

```diff
@@ -306,3 +306,15 @@
 # Prereg challenge data is uploaded to this project TODO: Delete when Prereg challenge ends
 PREREG_DATA_STORE_TOKEN = None
 PREREG_DATA_STORE_GUID = None
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
+        'LOCATION': 'osf_cache_table',
```

Comment:
Hmm, have you thought through the implications of changing the [default cache](https://docs.djangoproject.com/en/2.1/topics/cache/#local-memory-caching)? My first thought is that we would want to keep the default cache as `django.core.cache.backends.locmem.LocMemCache`.  I don't think [waffle](https://waffle.readthedocs.io/en/v0.11/starting/configuring.html#configuring-waffle) was the only item using this default cache, so now it looks like a lot of unintentional items are showing up in the DatabaseCache.  

other cache_keys:
<img width="242" alt="screen shot 2019-01-08 at 4 16 33 pm" src="https://user-images.githubusercontent.com/9755598/50862255-d43bfc80-1360-11e9-9cf9-83889317b47e.png">

Reply:
Any thoughts on this @Johnetordoff @sloria ?  What are the pros/cons of adding these extra items to the database cache instead of the LocMemCache?  Are there parts of the site we might slow down because we're fetching it from here?

Reply:
Update from discussing with Steve - sounds like having a database cache will be an improvement over the localmemcache, having a central place for things like throttling.  However, we do need to shorten the expiration on the storage cached items.